diff -ur /usr/src/linux-headers-6.6.62-current-sunxi/sound/soc/codecs/Kconfig /home/ladmin/pcm1808/Kconfig
--- /usr/src/linux-headers-6.6.62-current-sunxi/sound/soc/codecs/Kconfig	2024-11-25 12:13:29.000000000 +0000
+++ /home/ladmin/pcm1808/Kconfig	2025-02-02 13:08:56.000000000 +0000
@@ -167,6 +167,7 @@
 	imply SND_SOC_PCM1789_I2C
 	imply SND_SOC_PCM179X_I2C
 	imply SND_SOC_PCM179X_SPI
+	imply SND_SOC_PCM1808
 	imply SND_SOC_PCM186X_I2C
 	imply SND_SOC_PCM186X_SPI
 	imply SND_SOC_PCM3008
@@ -1305,6 +1306,9 @@
 	help
 	  Enable support for Texas Instruments PCM179x CODEC.
 	  Select this if your PCM179x is connected via an SPI bus.
+	  
+config SND_SOC_PCM1808
+	tristate "Texas Instruments PCM1808 ADC"
 
 config SND_SOC_PCM186X
 	tristate
diff -ur /usr/src/linux-headers-6.6.62-current-sunxi/sound/soc/codecs/Makefile /home/ladmin/pcm1808/Makefile
--- /usr/src/linux-headers-6.6.62-current-sunxi/sound/soc/codecs/Makefile	2024-11-25 12:13:29.000000000 +0000
+++ /home/ladmin/pcm1808/Makefile	2025-02-02 13:11:38.000000000 +0000
@@ -186,6 +186,7 @@
 snd-soc-pcm179x-codec-objs := pcm179x.o
 snd-soc-pcm179x-i2c-objs := pcm179x-i2c.o
 snd-soc-pcm179x-spi-objs := pcm179x-spi.o
+snd-soc-pcm1808-objs := pcm1808.o
 snd-soc-pcm186x-objs := pcm186x.o
 snd-soc-pcm186x-i2c-objs := pcm186x-i2c.o
 snd-soc-pcm186x-spi-objs := pcm186x-spi.o
@@ -569,6 +570,7 @@
 obj-$(CONFIG_SND_SOC_PCM1789)	+= snd-soc-pcm1789-codec.o
 obj-$(CONFIG_SND_SOC_PCM179X_I2C)	+= snd-soc-pcm179x-i2c.o
 obj-$(CONFIG_SND_SOC_PCM179X_SPI)	+= snd-soc-pcm179x-spi.o
+obj-$(CONFIG_SND_SOC_PCM1808)	+= snd-soc-pcm1808.o
 obj-$(CONFIG_SND_SOC_PCM186X)	+= snd-soc-pcm186x.o
 obj-$(CONFIG_SND_SOC_PCM186X_I2C)	+= snd-soc-pcm186x-i2c.o
 obj-$(CONFIG_SND_SOC_PCM186X_SPI)	+= snd-soc-pcm186x-spi.o
Only in /home/ladmin/pcm1808/: pcm1808.patch
